app/package.json

```diff
@@ -10,6 +10,7 @@
   },
   "license": "BSD-2-Clause",
   "dependencies": {
+    "keytar": "^4.0.3",
```

Comment:
Ah, no worries. This is added by `yarn`, when I `yarn add`ed. It's a shame that `yarn add` doesn't have a flag to pin it down exactly. See also https://github.com/yarnpkg/yarn/issues/1676